tests/view/observer/mutationobserver.js

```diff
@@ -267,6 +267,66 @@ describe( 'MutationObserver', () => {
 		expect( lastMutations[ 0 ].newText ).to.equal( 'xy' );
 	} );
 
+	// https://github.com/ckeditor/ckeditor5/issues/692 Scenario 1.
+	it( 'should handle space after inline filler at the end of container', () => {
+		const { view, selection } = parse( '<container:p>foo<attribute:b>[]</attribute:b></container:p>' );
+
+		viewRoot.appendChildren( view );
+		viewDocument.selection.setTo( selection );
+
+		viewDocument.render();
+
+		const inlineFiller = domEditor.childNodes[ 2 ].childNodes[ 1 ].childNodes[ 0 ];
+
+		inlineFiller.data += ' ';
+
+		mutationObserver.flush();
+
+		expect( lastMutations.length ).to.equal( 1 );
+		expect( lastMutations[ 0 ].type ).to.equal( 'children' );
+		expect( lastMutations[ 0 ].node ).to.equal( selection.getFirstPosition().parent );
```

Comment:
Shouldn't we check what's in the mutation? Would be more complete. Here and below.

Reply:
Unfortunately, FF will have bogus `br` element in children, while other browsers won't have it at all. I'm not sure whether it safe to check it. But I'll try to improve that part.

Reply:
I see. Don't try too hard then. If you won't come up with something in a couple of minutes just add a comment like your post above.